src/goMain.ts

```diff
@@ -109,7 +117,7 @@ export function activate(ctx: vscode.ExtensionContext): void {
 					revealOutputChannelOn: RevealOutputChannelOn.Never,
 					middleware: {
 						provideDocumentFormattingEdits: (document: vscode.TextDocument, options: FormattingOptions, token: vscode.CancellationToken, next: ProvideDocumentFormattingEditsSignature) => {
-							if (languageServerExperimentalFeatures['format'] === true) {
+							if (languageServerExperimentalFeatures['format']) {
```

Comment:
I specifically check for `true` here as user can provide anything in the settings and that value will make it through.

Reply:
I wasn't allowed to use a non-boolean value when testing, which is why thought this was possible, but maybe that's only a warning? Undid this in any case.